docs/conf.py

```diff
@@ -94,7 +93,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = ["_static"]
```

Comment:
Should there be a blank (comment) line before line 96, to match the ones at, e.g., line 90 and 105?